### PR TITLE
Remind new contributor PR

### PR DESCRIPTION
**Pull request by sticnarf**

### CODE REVIEW

File: bot/pkg/providers/pullstatus/notice.go [hunk 1]
```diff
@@ -95,3 +95,14 @@ func (p *pullStatus) noticeRedPacket(pull *github.PullRequest) error {
 	}
 	return nil
 }
+
+func (p *pullStatus) askForReviewer(pull *github.PullRequest) error {
+	if *pull.ReviewComments > 0 || len(pull.RequestedReviewers) > 0 {
+		return nil
```
Comment (you06) [inline bot/pkg/providers/pullstatus/notice.go]: For the PRs which already have some reviews but require asking for reviewer again will be returned here?

Comment (sticnarf) [reply bot/pkg/providers/pullstatus/notice.go]: Emm, yeah. There are some problems... Let me make it smarter...

Comment (sticnarf) [reply bot/pkg/providers/pullstatus/notice.go]: I call `ListReviews` API to get reviews. PTAL again...

File: bot/pkg/providers/pullstatus/notice.go [hunk 2]
```diff
@@ -95,3 +96,31 @@ func (p *pullStatus) noticeRedPacket(pull *github.PullRequest) error {
 	}
 	return nil
 }
+
+func (p *pullStatus) askForReviewer(pull *github.PullRequest) error {
+	reviewerIds := make(map[int64]struct{})
+	for _, reviewer := range pull.RequestedReviewers {
```
Comment (you06) [inline bot/pkg/providers/pullstatus/notice.go]: Like reviews, you can not access reviewers from pull request directly 🙃, call [`ListReviewers` ](https://pkg.go.dev/github.com/google/go-github/github?tab=doc#PullRequestsService.ListReviewers).

File: bot/pkg/providers/pullstatus/notice.go [hunk 3]
```diff
@@ -95,3 +96,31 @@ func (p *pullStatus) noticeRedPacket(pull *github.PullRequest) error {
...
+		reviewerIds[*reviewer.ID] = struct{}{}
+	}
+	if len(reviewerIds) >= 2 {
+		return nil
+	}
+
+	opt := github.ListOptions{}
+	reviews, _, err := p.opr.Github.PullRequests.ListReviews(context.Background(), p.owner, p.repo, pull.GetNumber(), &opt)
```
Comment (you06) [inline bot/pkg/providers/pullstatus/notice.go]: FYI, there are 3 types of reviews in a pull request, and out reviewers may use all the 3 types:
* [comments](https://pkg.go.dev/github.com/google/go-github/github?tab=doc#IssuesService.ListComments)
* [reviews](https://pkg.go.dev/github.com/google/go-github/github?tab=doc#PullRequestsService.ListReviews)
* [review comment](https://pkg.go.dev/github.com/google/go-github/github?tab=doc#PullRequestsService.ListReviewComments), eg: LGTM with approved only

We can improve this in the future.

**Pull request merged**